frontend/packages/console-app/src/components/nodes/NodeLogs.tsx

```diff
@@ -322,6 +352,8 @@ const NodeLogs: React.FC<NodeLogsProps> = ({ obj: node }) => {
       isWrapLines={isWrapLines}
       setWrapLines={setWrapLines}
       showSearch={!isLoadingLog && !errorExists}
+      selected={selected}
+      logLabel={logLabel}
```

Comment:
I don't think you need `logLabel` once you move `aria-label` to `MenuToggle`

Reply:
I believe it's still needed for this https://github.com/openshift/console/pull/14147/files#diff-124ac34ac858698f0d0d5912f9fb43c30d2d2e7f101348e3d1e36a30c2e0dd9cR135

Reply:
You don't need new props here since you have existing props that do what you want.